parachain/scripts/build-docker.sh

```diff
@@ -59,26 +59,15 @@ docker build ${NOCACHE_FLAG} --pull -f ./parachain/docker/Dockerfile \
     --build-arg https_proxy="$PROXY" \
     --add-host=host.docker.internal:host-gateway \
     --network host \
-    --target parachain \
-    -t ${GITUSER}/${GITREPO}:${TAG} .
-
-# Tag it with latest if no tag parameter was provided
-[ -z "$2" ] && docker tag ${GITUSER}/${GITREPO}:${TAG} ${GITUSER}/${GITREPO}:latest
-
-# Show the list of available images for this repo
-echo "Image is ready"
-echo "------------------------------------------------------------"
-docker images | grep ${GITREPO}
+    --target builder \
+    --tag local-builder:latest .
```

Comment:
I don't fully understand this - why do we need to build `local-builder` first? Or what's the advantage comparing to the current impl?

Reply:
In case we need to build all-in-one image, this can save the build time.
While splitting the original parachain image building into two steps doesn't create much overlap. We have this similar operation when building worker images, which works fie.